src/rpc/net.cpp

```diff
@@ -209,8 +208,8 @@ static RPCHelpMan getpeerinfo()
         if (stats.m_min_ping_usec < std::numeric_limits<int64_t>::max()) {
             obj.pushKV("minping", ((double)stats.m_min_ping_usec) / 1e6);
```

Comment:
unrelated note: Can use the count seconds double helper from #20044

Reply:
Thanks @MarcoFalke. I'll address this in the follow-up

Reply:
#21015 isn't merged yet, but hopefully soon will.

Reply:
I plan to review it soon

Reply:
I'll do this in #21198 once #21015 is merged.